Engine/DataFeeds/LiveTradingDataFeed.cs

```diff
@@ -409,8 +409,12 @@ protected Subscription CreateSubscription(Universe universe, Security security,
                     var enqueable = new EnqueueableEnumerator<BaseData>();
                     _customExchange.SetDataHandler(config.Symbol, data =>
                     {
-                        enqueable.Enqueue(data);
-                        if (subscription != null) subscription.RealtimePrice = data.Value;
+                        // Only enqueue new data if there is none (LastEnqueued == null) or if newer 
+                        if (enqueable.LastEnqueued == null || data.EndTime > enqueable.LastEnqueued.EndTime)
+                        {
```

Comment:
If custom data emits multiple points for the same moment in time will this hide those other points? Agree we need to trace higher to see why its emitting more often than it should